### PR TITLE
docs: add check server provision logs tutorial to docs

### PR DESCRIPTION
**Pull request by thlmenezes**
## Description of change

Created docs on checking server provision logs, including in the introduction guide; also accomplishes the "add action viewer logs tutorial to docs" issue

## Issues resolved by this PR

- [x] [DVN-10226]
- [x] [DVN-10227]

<details>
  <summary>More info to help validate your PR: links, images, videos, ...</summary>
  
image

</details>

[DVN-10226]: https://devopness.atlassian.net/browse/DVN-10226?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DVN-10227]: https://devopness.atlassian.net/browse/DVN-10227?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/servers/ssh-into-server.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ links:
     previous:
         - /docs/servers
     next:
-        - /docs/applications/add-application
+        - /docs/servers/check-server-provision-logs
```
Comment (jfoliveira) [inline docs/servers/ssh-into-server.md:10]: Next step of `SSH into server` is checking provision logs? 🤔 
Shouldn't user check provision logs right after provision a server, thus a way before SSH-ing into it?

File: docs/servers/check-server-provision-logs.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        - /docs/applications/add-application
+    guides:
+    related:
+    featured:
+---
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `Servers` card
+1. Click the `View` in the `Servers` card
+1. On the upper-right corner, click `ACTIONS` to see a list of actions for all `Servers`
```
Comment (jfoliveira) [inline docs/servers/check-server-provision-logs.md:19]:
```suggestion
1. On the upper-right corner, click `ACTIONS` to see a list of actions for all `Servers` of the current environment
```

File: docs/servers/check-server-provision-logs.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+1. In the list of actions, find the action you want to inspect and click `LOGS` to see the list of action steps
+    > There are 3 actions related to the server provision process: `Provision`, `Provision-check` and `Add`;
+    > each action can have one of the following status: `Pending`, `In progress`, `Completed` or `Failed`.
```
Comment (jfoliveira) [inline docs/servers/check-server-provision-logs.md:22]:
```suggestion
    > each action can have one of the following status: `Queued`, `Pending`, `In progress`, `Completed` or `Failed`.
```

File: docs/servers/check-server-provision-logs.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+1. In the list of action steps, find the action you want to inspect and click `LOGS`
```
Comment (jfoliveira) [inline docs/servers/check-server-provision-logs.md:23]:
```suggestion
1. In the action details view, find the server and the action step you want to inspect and click `LOGS`
```

File: docs/servers/check-server-provision-logs.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+1. In the card opened below the action step will be listed the logs related to that action step execution
```
Comment (jfoliveira) [inline docs/servers/check-server-provision-logs.md:24]: I believe the word `card` is being used here with a different meaning of the same word `card` used in steps 2 and 3, which could be misleading to the reader.
I'd suggest to be consistent and do not use the same word when referring to different UI elements.

```suggestion
1. A text area will be displayed containing the output log of the action step execution
```

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/servers/ssh-into-server.md (hunk 1 above)
Comment (thlmenezes) [reply docs/servers/ssh-into-server.md:10]: Will fix; I believe the best order for this post would be after adding a server and before ssh into it.

Comment (thlmenezes) [reply docs/servers/ssh-into-server.md:8]: Updated links for the introduction guide, now the current reading flow is:

```mermaid
flowchart TD
   A[add-aws-server] --> D[check-server-provision-logs]
   B[add-existing-server] --> D
   C[add-gcp-server] --> D
   D --> E[ssh-into-server]
```

**Review by jfoliveira — APPROVED** [review]

**Pull request merged**